cookbooks/mysql/recipes/install.rb

```diff
@@ -6,7 +6,7 @@
 known_versions = {
   # Note: mysql 5.5 is a limited access feature on this stack; use 5.6 or higher if possible.
   'dev-db/mysql' => ['5.5.49'],
-  'dev-db/percona-server' => ['5.6.28.76.1', '5.6.29.76.2-r1', '5.6.32.78.1', '5.6.35.81.0', '5.6.37.82.2', '5.6.44.86.0', '5.6.45.86.1', '5.7.13.6', '5.7.14.8', '5.7.17.13', '5.7.19.17', '5.7.26.29',  '5.7.27.30']
+  'dev-db/percona-server' => ['5.6.28.76.1', '5.6.29.76.2-r1', '5.6.32.78.1', '5.6.35.81.0', '5.6.37.82.2', '5.6.44.86.0', '5.7.13.6', '5.7.14.8', '5.7.17.13', '5.7.19.17', '5.7.26.29']
```

Comment:
@caesarprime Why did you revert this? I Was referring to the extra space character in this (original) line: 
```'5.7.26.29',  '5.7.27.30']``` Can you see it?

Reply:
@jfuechsl , my bad, it was accidentally made during a stash. Reverted back and made a commit with the amendments.